migration/migrator/data/course_tables.sql

```diff
@@ -1770,11 +1809,11 @@ ALTER TABLE ONLY public.course_materials_sections
 
 
 --
--- Name: course_materials_sections fk_section_id; Type: FK CONSTRAINT; Schema: public; Owner: -
```

Comment:
wait, why are we deleting/replacing a a constraint for course_materials_sections?